src/mondo_mapping_status.py

```diff
@@ -0,0 +1,82 @@
+"""Mapping status between Medgen and Mondo"""
+
+def medgen_mondo_mapping_status(
```

Comment:
Repurposed some GARD code for some mapping reports, e.g. `in_medgen` v `in_mondo` v `both`.